modyn/supervisor/internal/eval/strategies/abstract.py

```diff
@@ -19,9 +35,7 @@ def __init__(self, config: EvalStrategyConfig):
         self.config = config
 
     @abstractmethod
-    def get_eval_intervals(
-        self, first_timestamp: int, last_timestamp: int
-    ) -> Iterable[tuple[Optional[int], Optional[int]]]:
+    def get_eval_intervals(self, training_intervals: Iterable[tuple[int, int]]) -> Iterable[EvalInterval]:
```

Comment:
I guess I don't fully get the interface change yet. We used to pass the timestamps of the samples in the trigger. what do we pass now and why?

Reply:
For the `after_training` case the same, just wrapped in a list of length 1 of 2-tuples (start, end).
For the `after_pipeline` case, we pass the list of all of those tuple at once (that have been handled in `after_training` via multiple calls the course of the pipeline)